utilities/ideascale-importer/pyproject.toml

```diff
@@ -44,6 +44,9 @@ flake8 = "6.0.0"
 mypy = "0.991"
 mypy-extensions = "0.4.3"
 
+[tool.poetry.scripts]
+ideascale-importer = "ideascale_importer.app:app"
```

Comment:
I kept the `app` name, but it feels like something like `ideascale_importer.app:cli` could better describe what's going on.